uniswap_swap_detection/src/constant.ts

```diff
@@ -0,0 +1,35 @@
+import { ethers, getEthersProvider } from "forta-agent";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+const alchemyApiKey = process.env.ALCHEMY_API_KEY;
+export const UNISWAP_ADDRESS = "0x1F98431c8aD98523631AE4a59f267346ea31F984";
```

Comment:
```suggestion
export const UNISWAP_FACTORY_ADDRESS = "0x1F98431c8aD98523631AE4a59f267346ea31F984";
```
Suggestion to be more descriptive, which is always a good thing.